modules/cache/jvm/src/main/scala/coursier/cache/ConnectionBuilder.scala

```diff
@@ -18,7 +18,9 @@ import dataclass.data
   hostnameVerifierOpt: Option[HostnameVerifier] = None,
   method: String = "GET",
   maxRedirectionsOpt: Option[Int] = Some(20),
-  proxy: Option[Proxy] = None
+  proxy: Option[Proxy] = None,
+  @since("2.0.13")
```

Comment:
It shoud be `2.0.16` now.
```suggestion
  @since("2.0.16")
```